buildSrc/src/main/groovy/airbyte-docker.gradle

```diff
@@ -174,6 +174,7 @@ class AirbyteDockerPlugin implements Plugin<Project> {
 
     static def createTask(Project project, String taskName, String dockerFile) {
         if (project.file(dockerFile).exists()) {
+            // TODO -- why does this exclude venv files?
```

Comment:
Not an expert and I don't have a lot of context but this seems to assume that `.venv` is never copied by the Dockerfile and that there will be an `pip install` command in the Dockerfile to make the virtual environment available in the docker image/container (which seems like an assumption that isn't too far-fetched). 

What I'm missing is that we've done this assumption in order to solve a problem and I don't get what the problem was. My best guess is performance but I doubt this would be very costly not to ignore .venv